lib/menubar/init.lua

```diff
@@ -69,6 +69,8 @@ menubar.right_label = "▶▶"
 --- Label used for "Previous page", default "◀◀".
 menubar.left_label = "◀◀"
 
+local list_interspace = theme.xresources.apply_dpi(4) * 3 -- common.list_update padding
```

Comment:
Why not `apply_dpi(12)`? I guess this is about the padding that `common.list_update` adds around the textbox and imagebox, right? The comment should explain this better.